scripts/question_answering/config.py

```diff
@@ -0,0 +1,158 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=
+r"""
+This file contains all hyperparameters.
+"""
+import argparse
+
+import mxnet as mx
+
+parser = argparse.ArgumentParser(
+    description='The hyperparameters for QANet and optimizer.')

```

Comment:
Please specify `formatter_class=argparse.ArgumentDefaultsHelpFormatter`, so that the default values are included in the `--help` output